cmake/external/libfuzzer.cmake

```diff
@@ -32,14 +32,16 @@ ExternalProject_Add(
 
   DOWNLOAD_DIR ${FIREBASE_DOWNLOAD_DIR}
   DOWNLOAD_NAME libfuzzer
-  SVN_REPOSITORY https://llvm.org/svn/llvm-project/compiler-rt/trunk/lib/fuzzer
+  SVN_REPOSITORY "http://llvm.org/svn/llvm-project/compiler-rt/trunk/lib/fuzzer"
+  SVN_REVISION -r "338160"
```

Comment:
Where possible, I'd recommend you use a tag, rather than a revision. It communicates a bit more to the reader.

Reply:
ok I used the current tag.